servers/lib/server/handlers/findteam.coffee

```diff
@@ -0,0 +1,55 @@
+koding  = require './../bongo'
+async   = require 'async'
+Tracker = require '../../../../workers/social/lib/social/models/tracker.coffee'
```

Comment:
i believe `../models/tracker.coffee` should work, models folder is linked under `servers` folder